test/fixtures/accounts.js

```diff
@@ -119,7 +119,7 @@ const Account = stampit({
 		this.publicKey =
 			publicKey ||
 			randomstring
-				.generate({ charset: '0123456789ABCDE', length: 32 })
+				.generate({ charset: '0123456789ABCDE', length: 64 })
```

Comment:
For reference and explanation. This function generate a random hexadecimal string of 32 length. While two hex characters combined represent one byte, resulting a public a key of 16 bytes. While the protocol definition says it would be 256 bits means 32 bytes.

Reply:
@lsilvs I just noticed that we are not including `F` in the charset, hence not using the full character set to represent the random key. Can you please add that as well.

Reply:
Done